src/constants/constants.ts

```diff
@@ -92,22 +92,22 @@ export const METHODS: Record<Method, MethodData> = {
   },
 };
 
-export enum SUPPORTED_VERIFICATION_FUNCTION_STRINGS {
+export enum SUPPORTED_VERIFICATION_METHOD_STRINGS {
   KECCAK256_UTF8 = 'keccak256(utf8)',
   KECCAK256_BYTES = 'keccak256(bytes)',
 }
 
-export enum SUPPORTED_VERIFICATION_FUNCTION_HASHES {
+export enum SUPPORTED_VERIFICATION_METHOD_HASHES {
   HASH_KECCAK256_UTF8 = '0x6f357c6a',
   HASH_KECCAK256_BYTES = '0x8019f9b1',
 }
 
 export type SUPPORTED_VERIFICATION_FUNCTIONS =
```

Comment:
Shouldn't this be changed to below for consistency?

```suggestion
export type SUPPORTED_VERIFICATION_METHODS =
```

Reply:
Changing.